config.js

```diff
@@ -0,0 +1,6 @@
+const host   = process.env.HOST || 'localhost';
```

Comment:
This file was in your previous PR. So it should not be here again except it really has some changes.
Please do the following things:
1. Close this PR
1. Create a new branch in your fork repo
This branch should have last changes from master
1. Move current changes from this PR to the new branch (read about the `cherry-pick` command, it can be usefull in this case)
1. Create a new PR (from the new branch to your master) with only required changes